script.js

```diff
@@ -128,39 +130,34 @@ class MixOrMatch {
         });
     }
     getCardType(card) {
-        let cardFront = card.children[1];
-        return cardFront.children[cardFront.children.length-1].src;
+        return card.getElementsByClassName('card-value')[0].src;
     }
     canFlipCard(card) {
-        if(this.busy || this.matchedCards.includes(card) || card === this.cardToCheck)
-            return false;
-        return true;
+        return !this.busy && !this.matchedCards.includes(card) && card !== this.cardToCheck;
     }
 }
 
-class GameManager {

```

Comment:
I completely removed the GameManager since it does not actually store any contextual data, and is essentially a function, since every time you create a new one it runs a function.